packages/compass-components/src/components/accordion.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css } from '@emotion/css';
+import Icon from '@leafygreen-ui/icon';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useId } from '@react-aria/utils';
+
+const labelStyles = css({
+  padding: 0,
+  margin: 0,
+  fontWeight: 'bold',
+});
+
+const buttonStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  borderStyle: 'solid',
+  borderColor: 'transparent',
+  borderRadius: '6px',
+  borderWidth: '3px',
+  background: 'none',
+  '&:focus-visible': {
+    borderColor: uiColors.green.base,
+  },
+});
+const containerStyles = css({
+  marginTop: spacing[3],
+  display: 'flex',
+  alignItems: 'center',
+  '&:hover': {
+    cursor: 'pointer',
+  },
+});
+interface AccordionProps {
+  dataTestId?: string;
+  text: string;
+}
+function Accordion(
+  props: React.PropsWithChildren<AccordionProps>
+): React.ReactElement {
+  const [open, setOpen] = useState(false);
+  const regionId = useId('region-');
+  const labelId = useId('label-');
+  return (
+    <div data-testid={props.dataTestId}>
+      <div className={containerStyles}>
+        <p className={labelStyles} id={labelId}>
+          <button
+            className={buttonStyles}
+            type="button"
+            aria-expanded={open ? 'true' : 'false'}
+            aria-controls={regionId}
+            onClick={() => setOpen(!open)}
```

Comment:
Tiny nit that is not that important here, but might be helpful in general. If your state is derived from the current one, you might want to use callback as an argument for the setter:

```suggestion
            onClick={() => {
               setOpen(currentOpen => !currentOpen)
            }}
```

This gives better guarantees that the state will not be affected by async nature of the events that can activate them